src/DataCollector/MongoQuerySerializer.php

```diff
@@ -70,7 +74,7 @@ public static function prepareItemData($item)
             return $item->bsonSerialize();
         }
 
-        if (\is_array($item) || \is_object($item)) {
+        if (\is_object($item)) {
```

Comment:
If `|| \is_object($item)` is checked at line 57, this if is futile then.

Reply:
Yes I made a mistake while re-applying my changes in the fork. Updated line 57